docs/current_docs/api/return-values.mdx

```diff
@@ -47,6 +47,79 @@ The result will look like this:
 Linux buildkitsandbox 6.1.0-22-cloud-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.94-1 (2024-06-21) x86_64 x86_64 x86_64 GNU/Linux
 ```
 
+## Integer return values
+
+Here is an example of a Dagger Function that returns the sum of two integers:
+
+<Tabs groupId="language">
+<TabItem value="Go">
+```go file=./snippets/functions/return-values-integer/go/main.go
+```
+</TabItem>
+
+<TabItem value="Python">
+```python file=./snippets/functions/return-values-integer/python/main.py
+```
+</TabItem>
+
+<TabItem value="TypeScript">
+```typescript file=./snippets/functions/return-values-integer/typescript/index.ts
+```
+</TabItem>
+
+</Tabs>
+
+Here is an example call for this Dagger Function:
+
+```shell
+dagger call add-integer --a 1 --b 2
+```
+
+The result will look like this:
+
+```shell
+3
+```
+
+## Float return values
+
+:::warning
+Floats are currently only supported in Go and TypeScript SDKs for now.
```

Comment:
```suggestion
The information in this section is only applicable to the Go and TypeScript SDKs. Floating-point number arguments are not currently supported in the Go SDKs.
```

Reply:
Removed because now we have a Python snippet